spec/views/forms/index.html.erb_spec.rb

```diff
@@ -14,7 +16,7 @@
     end
 
     it "does not contain a a list of forms" do
-      expect(rendered).not_to have_text "Your forms"
+      expect(rendered).not_to have_table
```

Comment:
Is there any other content on the page that we could also check to ensure that its not returning a list of forms?

The problem with this test is that its will still pass if we change the content from using a table to was an `<ul>` or even summary card components and for some reason it outputs content to page.